Readme.md

```diff
@@ -46,23 +48,17 @@ Don't hesitate to [reach-out](#Reaching-out) if you are not able to use our prof
 
 ### Safety
 
-Unlike runtime profilers, the native profiler requires no code modifications of the target service.  It doesn't direct signals at the target, use any `LD_PRELOAD` tricks, replace shared objects, or otherwise interfere with program execution at the process level once the target application has been launched.
-
-In particular:
-
-- While segfaults and deadlocks can interrupt profiling, they do not propagate to the target application.
-- PID wrapper returns the PID of the target, rather than the PID of `ddprof`.  This is great when you are already running your target under a wrapper or if you're trying to wrap the init process of a PID namespace (as might be the case for containers).
-- ddprof isolates its memory footprint to restartable sub-processes
-- ddprof schedules its work on a single CPU core
+- Most operations are performed out of process
```

Comment:
Right, good addition!  I think we should specify/clarify a bit more, since "out of process" has also grown to take on the same technical meaning as "out of core" from TAOCP and similar works.  Proposing an idea even though I'm slightly unhappy with my own suggestion.  🙃